packages/js/product-editor/src/components/header/woo-more-menu-item/Readme.md

```diff
@@ -0,0 +1,73 @@
+# 
+
+The `WooProductMoreMenuItem` slot allows developers to add custom items to the "More" menu
+in the WooCommerce Product Editor header.
+
+![Product editor - More Menu](image-product-editor-more-menu-1)
```

Comment:
This is not pointing anywhere valid

Reply:
correct. It's going to be used to update the blog post in https://developer.woocommerce.com/